.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Release
+
+permissions:
+  id-token: write
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    name: Perform Publishing
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Use Python 3.11
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Install
+        shell: bash
+        run: |
+          yarn global add node-gyp
+          yarn --skip-integrity-check --network-timeout 100000
+        env:
+          NODE_OPTIONS: --max_old_space_size=4096
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # https://github.com/microsoft/vscode-ripgrep/issues/9
+
+      - name: Publish NPM
+        shell: bash
+        run: |
+          yarn publish:latest
```

Comment:
Last time I ran the release `yarn publish:latest` was an interactive command. 
I had to select the incremented version, which is then written to the package.json files. I think we might have to pass this via an argument to the workflow?
Also `packages/core/README.md` is update and the package.json changes have to be committed: 
https://github.com/eclipse-theia/theia/blob/master/doc/Publishing.md#release

I think the translation workflow opens PRs with changes atm (https://github.com/eclipse-theia/theia/blob/master/.github/workflows/translation.yml), so the required changes might be similar

Reply:
I see, thanks. I've updated the PR to make this into a workflow dispatch:
1. Allows to select minor/patch version for release.
2. After the publishing is done, it will automatically create a PR from the changes.